Modules/CustomRolesHelper.cs

```diff
@@ -1061,8 +1062,7 @@ public static bool CheckAddonConfilct(CustomRoles role, PlayerControl pc)
                     || pc.Is(CustomRoles.Vindicator) 
                     || pc.Is(CustomRoles.TicketsStealer)
                     || pc.Is(CustomRoles.Pickpocket) 
-                    || pc.Is(CustomRoles.Glitch)
-                    || pc.Is(CustomRoles.Dictator))
+                    || pc.Is(CustomRoles.Glitch))

```

Comment:
Why Dictator was removed?

Reply:
Dictator's skip is also counted and VoidBallot won't influence dictator's skill.

Reply:
Then this Add-on is simply useless for him, although he could get a more useful add-on

Reply:
Dictator should not be able to be Void Ballot.